### PR TITLE
Gazelle 18 (gaze18)

### PR DESCRIPTION
**Pull request by jacobgkau**
This PR adds the Gazelle 18 (gaze18) to tech docs.

### CODE REVIEW

File: src/models/gaze18/internal-overview.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Webcam/microphone connector is highlighted in yellow
+- Speaker connectors are highlighted in cyan
+- Main battery connector is highlighted in orange
+- Wireless card is highlighted in blue
+- RAM is highlighted in maroon
+- BIOS flash chip (U24) is highlighted in black
+- Touchpad connector is highlighted in white
+- M.2 SSDs are highlighted in dark green
+    - Innermost slot supports PCIe NVMe Gen 4
+    - Outermost slot supports PCIe NVMe Gen 3
```
Comment (ahoneybun) [inline src/models/gaze18/internal-overview.md:19]: Do we want to note Gen 3 or SATA?

File: src/models/gaze18/repairs.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+2. Pull the small tabs on both sides of the RAM simultaneously. The RAM should spring up to an angle.
+
+![RAM slots](./img/ram-slots.webp)
+
+3. Remove the RAM from the slot.
+4. Insert the new RAM (or reseat the existing RAM) by placing it in the keyed slot and pressing down until it clicks into place.
+
+## Replacing an M.2/NVMe SSD:
+
+This model supports up to two M.2 SSDs. Both M.2 slots are size 2280. The inner-most slot supports PCIe NVMe Generation 4, and the outer-most slot supports PCIe NVMe Generation 3.
```
Comment (ahoneybun) [inline src/models/gaze18/repairs.md:53]: Same note about SATA support.

**Review by ahoneybun — DISMISSED** [review]
Looks great just a few small things that are totally optional.

File: src/models/gaze18/internal-overview.md (hunk 1 above)
Comment (jacobgkau) [reply src/models/gaze18/internal-overview.md:19]: It doesn't support SATA, so no.

File: src/models/gaze18/repairs.md (hunk 2 above)
Comment (jacobgkau) [reply src/models/gaze18/repairs.md:53]: Doesn't support SATA, only NVMe Gen 3.

**Comment by ahoneybun** [comment]
It says that it supports SATA:

- Storage
    - 1x M.2 (PCIe NVMe Gen 4)
    - 1x M.2 (PCIe NVMe Gen 3 or SATA)
    - MicroSD card reader

**Comment by jacobgkau** [comment]
> It says that it supports SATA:

That was the issue. Good catch, fixed.

**Review by ahoneybun — APPROVED** [review]
LGTM :tada:

**Pull request merged**